client/src/common/asyncComputed.ts

```diff
@@ -4,13 +4,13 @@
 
 import { computed, isRef, ref, Ref, shallowRef, watchEffect } from 'vue';
 
-export type AsyncComputedOnCancel = (cancelCallback: any) => void
+export type AsyncComputedOnCancel = (cancelCallback: any) => void;
 
 export interface AsyncComputedOptions {
-  lazy?: boolean
-  evaluating?: Ref<boolean>
-  shallow?: boolean
-  onError?: (e: unknown) => void
+  lazy?: boolean;
+  evaluating?: Ref<boolean>;
+  shallow?: boolean;
+  onError?: (e: unknown) => void;
 }
```

Comment:
Can we update prettier rules to avoid this horendous format style for interfaces?

Reply:
I haven't seen anything for that. I kinda like it this way but I'm a C and C++ guy, I like `;` after my attributes.
Prettier configuration is very minimal, which is a good thing, we don't want too many options.

Reply:
LGTM